tests/foreman/api/test_permission.py

```diff
@@ -264,7 +258,10 @@ def set_taxonomies(self, entity, organization=None, location=None):
         return entity
 
     @pytest.mark.tier1
-    def test_positive_check_create(self):
+    @pytest.mark.parametrize(
+        'entity', **parametrized([entities.Architecture, entities.Domain, entities.ActivationKey])
+    )
+    def test_positive_check_create(self, entity):
```

Comment:
I would use `entity_cls` for the parametrization here, so that its clear its not an instance of an entity.